keel-web/src/main/kotlin/com/netflix/spinnaker/config/KeelConfigurationFinalizer.kt

```diff
@@ -63,6 +65,25 @@ class KeelConfigurationFinalizer(
       }
   }
 
```

Comment:
cool that we now have a way to register this stuff here instead of what we were doing before!

Reply:
I still don't know why this project isn't using `ObjectMapperSubtypeConfigurer`... It's far less code to maintain from your side of things, and is a lot more flexible than this strategy.

Reply:
I'll defer to @robfletcher on this one.